### DISCUSSION TITLE
Disable CSRF

### DISCUSSION THREAD

**Discussion by IgorDePaula** [thecodingmachine/graphqlite#492] [General]
Why disable CSRF and not use api routes? Api routes not have csrf tokens.

**Comment by IgorDePaula** [comment]
I means in Laravel.

**Comment by oojacoboo** [comment]
If this issue is in regards to Laravel, please direct the issue to the Laravel repo:

https://github.com/thecodingmachine/graphqlite-laravel

I'm not aware of any CSRF disabling that's done within GraphQLite.  Can you please provide an example or point to the code that's an issue?
